utils/utils.go

```diff
@@ -0,0 +1,25 @@
+package utils
```

Comment:
I don't like the `utils` package but unclear what a better name could be. We should at least une `internal/utils` so that it remains outside of the public API of `firehose-core`.